.tekton/integration-tests.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: upstream integration tests
+spec:
+  description: >-
+    TBD
```

Comment:
Could we add a description here so there's context for those that will come to see this file later?

Reply:
Sure. Can you help me with crafting this field content?

Reply:
What is the context?

Reply:
This pipeline is invoking the e2e tests. The description might include the e2e structure or/and what it is testing.

Reply:
Okay - you would know better than I so if you can add something about that I can review it!